base/system/winlogon/lang/id-ID.rc

```diff
@@ -19,7 +19,7 @@ FONT 8, "MS Shell Dlg"
 BEGIN
     LTEXT "DLL Antarmuka Masuk Pengguna %s gagal diambil.", IDC_GINALOADFAILED, 39, 16, 156, 24
     LTEXT "Hubungi administrator sistem anda untuk mengganti DLL, atau mengembalikan DLL aslinya.", IDC_GINALOADFAILEDCONTACT, 39, 53, 151, 25
-    DEFPUSHBUTTON "&Mulai Lagi", 1, 80, 91, 68, 14
+    DEFPUSHBUTTON "&Mulai Ulang", IDOK, 80, 91, 68, 14
```

Comment:
This makes the translation the same as in the `IDD_SYSSHUTDOWN` dialog below, and also, it appears to be the preferred translation by Windows folks as well:
https://support.microsoft.com/id-id/windows/mulai-ulang-boot-ulang-pc-anda-110262aa-fc79-1c33-7b00-c140ae3a6dac